### PR TITLE
CTP: RawDataReader params changesL filter added

### PR DESCRIPTION
**Pull request by lietava**
1. CTP: RawDataReader params changesL filter added - no filter needed in qc 
2. New CTP config parser
3 Check for two part zmq message in ctp-proxy removed
4. Minor modifications of counters related code

### CODE REVIEW

**Comment by lietava** [comment]
Hi @shahor02 : please, check when possible. Cheers, R.

File: Detectors/CTP/reconstruction/src/RawDataDecoder.cxx [hunk 1]
```diff
@@ -273,13 +273,16 @@ int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::map<o2::I
...
+      addCTPDigit(linkCRU, rdhOrbit, remnant, pldmask, digitsMap);
       LOG(debug) << "diglet:" << remnant << " " << (remnant & bcmask).to_ullong();
       remnant = 0;
     }
   }
   if (mDoLumi) {
     lumiPointsHBF1.emplace_back(LumiInfo{orbit0, 0, 0, countsMBT, countsMBV});
+    for (auto const& dig : digitsMap) {
+      digits.push_back(dig.second);
+    }
```
Comment (shahor02) [inline Detectors/CTP/reconstruction/src/RawDataDecoder.cxx:287]: not sure about this: you are pushing digits w/o checking if they are requested?

Comment (lietava) [reply Detectors/CTP/reconstruction/src/RawDataDecoder.cxx:287]: Fixed.

**Comment by lietava** [comment]
@shahor02 : it does not seem to be ctp error ?

**Review by shahor02 — APPROVED** [review]

**Pull request merged**